src/main/java/core/basesyntax/db/FruitShopDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface FruitShopDao {
+    int storageSize();
+
+    List<String> getKeyAndValue();
+
+    void put(String fruit, int quantity);
```

Comment:
The method 'put' should have a more descriptive name to indicate what it actually does. For instance, 'addOrUpdateFruitQuantity' would better describe the action of putting a fruit with its quantity into the storage.